addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/mobile_search_panel/mobile_search_panel.xml

```diff
@@ -18,7 +18,7 @@
                                 </header>
                                 <ul class="list-group d-block o_search_panel_field">
                                     <li t-foreach="group.dashboards" t-as="dashboard" t-key="dashboard.id" t-on-click="() => this.onDashboardSelected(dashboard.id)" class="o_search_panel_category_value list-group-item py-1 o_cursor_pointer border-0 ps-0 pe-2">
-                                        <header class="list-group-item list-group-item-action d-flex align-items-center p-0 border-0" t-att-class="{'active text-900 fw-bold': dashboard.id === props.activeDashboard.id}">
+                                        <header class="list-group-item list-group-item-action d-flex align-items-center p-0 border-0" t-att-class="{'active text-900 fw-bold': dashboard.id === props.activeDashboard?.id}">
```

Comment:
the layout seems broken in mobile mode :/
<img width="718" alt="image" src="https://github.com/odoo/odoo/assets/10333790/b14430fb-0024-4386-943e-ef1b095b175e">

Reply:
Mmm on my side it's correct :/ Did you refresh the page ?
![image](https://github.com/odoo/odoo/assets/30048414/f9b8518d-8e5e-49d7-b503-4db50faf21ad)